lib/frozen_record/backends/yaml.rb

```diff
@@ -56,19 +56,19 @@ def load(file_path)
 
       if supports_freeze
         def load_file(path)
-          YAML.load_file(path, freeze: true) || Dedup::EMPTY_ARRAY
+          YAML.unsafe_load_file(path, freeze: true) || Dedup::EMPTY_ARRAY
```

Comment:
You can't assume these methods exists. They were added very recently, so you have to add some `respond_to?` checks.

Other than that yes, this is the way to go. In-repo YAML files are trusted.

Reply:
added